src/warc2zim/content_rewriting/html.py

```diff
@@ -206,6 +207,8 @@ def process_attr(
             return (attr_name, ", ".join(new_value_list))
         if attr_name == "style":
             return (attr_name, self.css_rewriter.rewrite_inline(attr_value))
+        if attr_name.startswith("on") and attr_name[2] != "-":
```

Comment:
Won't that crash if atr name is just "on"?

Reply:
Indeed. Added testcases and adapted code with `if attr_name.startswith("on") and not attr_name.startswith("on-")` (probably even easier to read anyway)